docs/governance/staking-with-realms.md

```diff
@@ -1,82 +1,69 @@
-pagination_label: Realms
-sidebar_label: Staking veHNT on Realms
-description: Helium on Realms Documentation
+id: staking-with-helium-vote
+title: Staking on Helium Vote
+pagination_label: Helium Vote
+sidebar_label: Staking on Helium Vote
+description: Helium on Helium Vote Documentation
 image: https://docs.helium.com/img/link-image.png
-slug: /governance/staking-with-realms
+slug: /governance/staking-with-helium-vote
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-<img className="docsheader" src={useBaseUrl('/img/blockchain/realmsheader.png')} />
-
-Like many other Solana Ecosystem projects, Helium utilizes [Realms](https://realms.heliumvote.com/)
-to organize and manage vote escrow tokens, delegation, and more. Stakes can be created and later
-managed within Realms at any time at https://realms.heliumvote.com/.
+Helium utilizes [Helium Vote](https://heliumvote.com), a native application built by the Helium 
+Foundation to organize and manage vote escrow tokens, delegation, and more. Stakes can be created 
+and later managed within Helium Vote at any time at https://heliumvote.com.
 
 ## Creating a Stake
 
-Anyone with HNT in a wallet can create a stake using Realms either using an in-wallet browser or a
-browser-connected wallet.
+Anyone with HNT, MOBILE or IOT in their Helium Wallet App or Solana-compatible browser wallet
+can be used (such as Phantom orBackpack).
 
 While staking IOT or MOBILE tokens is possible through their respective subDAOs, those positions are
 only eligible for voting power and cannot currently be delegated for the purposes of earning
 rewards.
 
-This guide outlines using Realms with the [Helium Wallet App](/wallets/helium-wallet-app). To use
-Realms on a desktop browser, a Solana-compatible browser wallet can be used (such as Phantom or
-Backpack).
+This guide outlines using Helium Vote with the [Helium Wallet App](/wallets/helium-wallet-app). To use
+Helium Vote on a desktop browser, a Solana-compatible browser wallet can be used (such as Phantom or
+Backpack) by navagating to https://heliumvote.com.
 
-The Helium Wallet App's browser can be accessed via the globe icon in the bottom navigation.
 
-<figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/realmsbrowser.png')} />
-  <figcaption>
-    veHNT can be managed via <code>https://realms.heliumvote.com/dao/hnt</code>
-  </figcaption>
-</figure>
-<br />
+### Staking in the Helium Wallet App
 
-### Signing Into Realms
-
-Connect your account to Realms using the in-app browser from the Helium Wallet App.
+Staking in the Helium Wallet App can be done by navigating to the <code>Governance</code> tab within the Helium Wallet.
+Ensure you select the token you want to stake (HNT, MOBILE or IOT) at the top.
 
 <figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/connectaccount.png')} />
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933058646')} />
```

Comment:
add images to the /static/img folder in docusaurus

Reply:
@jthiller I'm a novice when it comes to coding and GitHub. Can you direct me to or provide the link to that folder in docusaurus?